noetic/distribution.yaml

```diff
@@ -2483,6 +2483,22 @@ repositories:
       url: https://github.com/ethz-adrl/ifopt.git
       version: master
     status: maintained
+  ifxbgt60dev:
+    doc:
+      type: git
+      url: https://github.com/Yang-IFX/ifxbgt60dev.git
+      version: master
+    release:
```

Comment:
You don't appear to have made a release. This whole stanza should not be included. bloom will generate it when you make a release following the first time release tutorial http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease